MAUI/DataForm/Data-annotations.md

```diff
@@ -0,0 +1,368 @@
+
+{% tabs %}
+{% highlight Bindable attribute}
+
+    [Bindable(false)]
+    public string Name { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Editable attribute
+
+It specifies whether the data field is editable or not.
+
+{% tabs %}
+{% highlight Editable attribute}
+
+    [Editable(false)]
+    public string Name { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+## ReadOnly attribute
+
+It specifies whether the data field is read only or not.
+
+{% tabs %}
+{% highlight ReadOnly attribute}
+
+    [ReadOnly(false)]
+    public string Name { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+Note: `ReadOnlyAttribute` takes higher priority than `EditableAttribute`
+
+## DataType attribute
+
+It specifies data type for the field.
+
+Supported data types are Text, MultilineText, Date, DateTime and Time.
+
+{% tabs %}
+{% highlight DataType attribute}
+
+    [DataType(DataType.MultilineText)]
+    public string Address { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Custom attribute
+The data form supports the following custom attribute, and these attributes can be accessible using `Syncfusion.Maui.DataForm`.
+
+### DataFormDisplayOptions attribute
+
+<table>
+<tr>
+<th>
+Property
+</th>
+<th>
+Details
+</th>
+</tr>
+<tr>
+<td>
+[RowSpan]
+</td>
+<td>
+Specifies the row span for the data form item.
+</td>
+</tr>
+<tr>
+<td>
+[ColumnSpan]
+</td>
+<td>
+Specifies the column span for the data form item.
+</td>
+</tr>
+<tr>
+<td>
+[ValidMessage]
+</td>
+<td>
+Specifies positive message to be shown when validation is passed. 
+</td>
+</tr>
+<tr>
+<td>
+[ShowLabel]
+</td>
+<td>
+Specifies whether the label should be visible or not.
+</td>
+</tr>
+<tr>
+<td>
+[ItemsOrderInRow]
+</td>
+<td>
+Specifies the ItemsOrderInRow property to display multiple editors in a single row. Two items with the same RowOrder property value are placed in this row according to their ItemOrderInRow property values.
+</td>
+</tr>
+</table>
+
+{% tabs %}
+{% highlight DataFormDisplayOptions attribute}
+
+    [DataFormDisplayOptions(Rowspan = 2, ColumnSpan = 2, RowOrder = 1, ItemsOrderInRow = 0)]
+    public string Name { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+### DataFormValueConverter attribute
+
+Specifies the Converter type which converts the original value in different format or as different value.
+
+{% tabs %}
+{% highlight  attribute}
+
+    [DataFormValueConverter(typeof(StringToDateConverter))]
+    public string Name { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+Refer to this`[link]` to know about `DataFormValueConverterAttribute`
+
+### DateFormDateRange attribute
+
+<table>
+<tr>
+<th>
+Property
+</th>
+<th>
+Details
+</th>
+</tr>
+<tr>
+<td>
+[MaximumDate] 
+</td>
+<td>
+Specifies the required maximum date.
+</td>
+</tr>
+<tr>
+<td>
+[MinimumDate] 
+</td>
+<td>
+Specifies the required minimum date.
```

Comment:
Specifies the minimum date that can be selected in date editor.

Reply:
content updated